apps/organisations/templates/organisation_landing_page.html

```diff
@@ -18,9 +18,9 @@
         <div class="row">
             <div class="l-col-6-0">
                 <h1 class="u-no-margin marked-box">{{ organisation.title }}</h1>
-                <h2 class="u-no-margin organisation__slogan">{{ organisation.slogan }}</h2>
+                <h3 class="u-no-margin organisation__slogan">{{ organisation.slogan }}</h3>
```

Comment:
should this even be a h anything? not sure if it counts as a heading at all to be honest?

Reply:
Yeah, I think you're right, it shouldn't. Hmm, or like as a sub-headline?